### PR TITLE
Fix label name with long text

### PR DESCRIPTION
**Pull request by anas-drissi**

### CODE REVIEW

File: src/components/directory-breadcrumbs.js [hunk 1]
```diff
@@ -66,6 +67,11 @@ const DirectoryBreadcrumbs = () => {
     const selectedDirectory = useSelector((state) => state.selectedDirectory);
     const currentPath = useSelector((state) => state.currentPath);
 
+    function limitChar(str, limit) {
+        return str.length > limit ? `${str.slice(0, limit)}...` : str;
+    }
+    const limitElementNameLength = 160;
```
Comment (laurgarn) [inline src/components/directory-breadcrumbs.js]: Defining it before limitChar (mostly for clarity), you could have used it directly in limitChar, avoiding repeating it in the calls.

Comment (anas-drissi) [reply src/components/directory-breadcrumbs.js]: it's fixed

File: src/components/dialogs/delete-dialog.js [hunk 2]
```diff
@@ -26,6 +28,11 @@ import CircularProgress from '@mui/material/CircularProgress';
  * @param {String} simpleDeleteFormatMessageId Format message id for simple delete
  * @param {String} error Error message
  */
+const useStyles = makeStyles((theme) => ({
+    tooltip: {
+        maxWidth: '1000px',
```
Comment (EstherDarkish) [inline src/components/dialogs/delete-dialog.js:33]: theme.spacing(125) instead ?

File: src/components/dialogs/delete-dialog.js [hunk 3]
```diff
@@ -109,7 +124,25 @@ const DeleteDialog = ({
                     </Grid>
                     {items.slice(0, 10).map((file) => (
                         <Grid item key={file.elementUuid}>
-                            <span> {file.elementName} </span>
+                            <span>
+                                {' '}
```
Comment (EstherDarkish) [inline src/components/dialogs/delete-dialog.js]: {' '} should be removed.

File: src/components/dialogs/delete-dialog.js [hunk 4]
```diff
@@ -109,7 +124,25 @@ const DeleteDialog = ({
...
+                                    >
+                                        {
+                                            <OverflowableText
+                                                text={file.elementName}
+                                                style={{ width: '100%' }}
+                                                tooltipStyle={classes.tooltip}
+                                            />
+                                        }
+                                    </div>
+                                }{' '}
```
Comment (EstherDarkish) [inline src/components/dialogs/delete-dialog.js]: {' '} should be removed here too.

File: src/components/directory-content.js [hunk 5]
```diff
@@ -84,6 +86,9 @@ const useStyles = makeStyles((theme) => ({
     centeredCircularProgress: {
         alignSelf: 'center',
     },
+    tooltip: {
+        maxWidth: '1000px',
```
Comment (EstherDarkish) [inline src/components/directory-content.js:90]: theme.spacing(125) instead ?

File: src/components/dialogs/delete-dialog.js (hunk 3 above)
Comment (anas-drissi) [reply src/components/dialogs/delete-dialog.js]: it's fixed

**Review by EstherDarkish — APPROVED** [review]

**Review by laurgarn — APPROVED** [review]
Works

**Pull request merged**